executors.go

```diff
@@ -350,8 +350,8 @@ func (c *Connection) Update(model interface{}, excludeColumns ...string) error {
 			}
 
 			tn := m.TableName()
-			cols := columns.ForStructWithAlias(model, tn, m.As)
```

Comment:
This uses the outer `model` instead of the iteration model. Not sure if this was a bug or intended behavior but I assume it should have been the current model within the iteration.

Reply:
Yes, I think this has had problems in the past where `TableName()` had to be defined on the slice type or something similar. I agree that this should be `m` and not `model`. Can you please add a failing test case for this?

Reply:
It looks like it actually does not matter. It is not possible to pass a `interface{}` struct, because that breaks reflection everywhere. So only slices of a struct type are allowed, and it also makes sense to only allow them. Therefore it resembles the exact same behavior as before in my opinion.

Reply:
I know too little about reflection to judge this statement, so I would prefer reverting this to the original behavior just in case this breaks something somewhere.

Reply:
OK, reverted